azkaban-web-server/src/main/resources/azkaban/webapp/servlet/velocity/javascript.vm

```diff
@@ -29,7 +29,8 @@
      * used by all the views whenever APIs need to be called. Those wrapper functions should
      * apply application related parameters common to all requests such as this header.
     *#
-  $.ajaxSetup({
+  #set ( $d = "$" )
+  ${d}.ajaxSetup({
```

Comment:
Alternative escaping method:
```
#[[$]]#.ajaxSetup({
```
IMO it's uglier, but works too, if you/we/anyone prefers this form -- please let me know.